wallet/chainntfns.go

```diff
@@ -14,9 +14,16 @@ import (
 	"github.com/decred/dcrutil"
 	"github.com/decred/dcrwallet/chain"
 	"github.com/decred/dcrwallet/waddrmgr"
+	"github.com/decred/dcrwallet/wallet/txauthor"
 	"github.com/decred/dcrwallet/wtxmgr"
 )
 
+const (
+	// ticketPurchaseAttemptsPerBlock is the default number of ticket
+	// purchase attempts per block.
+	ticketPurchaseAttemptsPerBlock = 10
```

Comment:
To try to more evenly spread tickets out over time. On a competitive network with many users buying tickets, it probably makes more sense to have a lower number for automatic purchase because it's unlikely one wallet will have >50% of the tickets that round and there are 20 tickets per block max.